storage/flush.go

```diff
@@ -53,39 +52,25 @@ type flushManager struct {
 func newFlushManager(database database, scope tally.Scope) databaseFlushManager {
 	opts := database.Options()
 	return &flushManager{
-		database:  database,
-		opts:      opts,
-		nowFn:     opts.ClockOptions().NowFn(),
-		blockSize: opts.RetentionOptions().BlockSize(),
-		pm:        opts.PersistManager(),
-		status:    scope.Gauge("flush"),
+		database: database,
+		opts:     opts,
+		nowFn:    opts.ClockOptions().NowFn(),
+		pm:       opts.PersistManager(),
+		status:   scope.Gauge("flush"),
 	}
 }
 
-func (m *flushManager) NeedsFlush(t time.Time) bool {
+func (m *flushManager) NeedsFlush(start time.Time, end time.Time) bool {
```

Comment:
that link was for the `databaseNamespace.NeedsFlush()`, also added to `databaseFlushManager.NeedsFlush()` comment